lightning/src/routing/router.rs

```diff
@@ -5505,7 +5505,7 @@ pub(crate) mod bench_utils {
 	}
 }
 
-#[cfg(all(test, feature = "_bench_unstable", not(feature = "no-std")))]
+#[cfg(all(bench, not(features = "no-std")))]
 mod benches {
```

Comment:
The original is incorrect is it not? These benches will never be run because we run benches with "std" enabled since it is enabled by default (`RUSTC_BOOTSTRAP=1 RUSTFLAGS='--cfg=bench' cargo bench`).

Reply:
oh I wrote this comment on an old version of the PR, now it uses `#[cfg(bench)]`.